packages/react-avatar/.eslint.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I couldn't commit the merge changes if I didn't replace `tslint.json` with `.eslint.json` 😕